COMDOX-1472 Onboarding 2: Rewrites (#855)
commerce-docs/microsite-commerce-storefront: src/content/docs/get-started/backends.mdx

import Options from '@components/Options.astro';
import Option from '@components/Option.astro';
import TableWrapper from '@components/TableWrapper.astro';
import Term from '@components/Term.astro';

The storefront supports three backend types. Each type has its own prerequisites, so follow only the path that matches your project. Choosing the wrong type can lead to misleading signals—GitHub and DA.live may look healthy while previews or drop-ins fail—and may result in installing packages your license does not require.

For authoring storefront pages (Document Authoring, Universal Editor), see [Document Authoring Quick Start](/merchants/quick-start/document-authoring/). This page covers the Commerce backend the storefront connects to.

> Drop-in components — B2C (business-to-consumer, meaning typical shopper experiences) and B2B (business-to-business, meaning company accounts, quotes, and purchase orders) — require an Adobe Commerce as a Cloud Service or Adobe Commerce Optimizer license. PaaS customers who add an Adobe Commerce Optimizer license are covered, and that license includes drop-in usage. There is no separate “B2B backend” type.

Reviewer: ```suggestion
Drop-in components — B2C (business-to-consumer, meaning typical shopper experiences) and B2B (business-to-business, including company accounts, negotiable quotes, and purchase orders) — require an Adobe Commerce as a Cloud Service or Adobe Commerce Optimizer license. PaaS customers who add an Adobe Commerce Optimizer license are covered, and that license includes drop-in usage. There is no separate “B2B backend” type.
```

Backends fall into three categories:

- Commerce PaaS — Existing Adobe Commerce on Cloud infrastructure or on-premises deployment without an Adobe Commerce as a Cloud Service or Adobe Commerce Optimizer license
- Adobe Commerce as a Cloud Service — Cloud Service deployment whose license includes Storefront
- Adobe Commerce Optimizer — New Adobe Commerce Optimizer deployment, or an existing PaaS deployment that added Adobe Commerce Optimizer

<Aside type="tip" title="Related documentation">
This page covers Commerce backends only. For Adobe Commerce product help (Admin, deployments, merchandising services, integrations), start from <Link href={"https://experienceleague.adobe.com/en/docs/commerce"} text={"Adobe Commerce documentation"} /> on Experience League.

> For pages and blocks on da.live, see <Link href={"https://docs.da.live/"} text={"Document Authoring documentation"} /> or [Document Authoring Quick Start](/merchants/quick-start/document-authoring/).

Reviewer: Should da.live be in backticks?

Author: Thinking about it. Will save it for another PR for proper name usage.

</Aside>

## Which backend do you have?

<TableWrapper nowrap={[0]}>

| Feature | Commerce PaaS | Adobe Commerce as a Cloud Service | Adobe Commerce Optimizer |
|---------|---------------|-----------------------------------|--------------------------|
| License includes Storefront | No | Yes | Yes |
| Setup | Manual install required | Fully automated | Fully automated |

> | Version | v2.4.7+ self-managed | Auto-managed | Fully managed SaaS |

Reviewer: what does auto-managed mean? It's SaaS

## Backend topology

> This diagram focuses on the backend layer. It adds two optional pieces you might use during a phased move: <Term>Luma Bridge</Term> and the Adobe Commerce Optimizer Connector. Luma is Adobe's classic server-side Commerce storefront theme. If you run a <Term>Luma</Term> storefront today and are moving to Edge Delivery Services, Luma Bridge is a PHP module on Commerce so Luma pages and EDS drop-ins can share the same cart and sign-in session during the transition. For the full stack (drop-ins, boilerplate, and how services connect), see [Commerce services and backends](/get-started/architecture/commerce-services-and-backends/).

Reviewer: Did I mention this in a previous PR?

```suggestion
This diagram focuses on the backend layer. It adds two optional pieces you might use during a phased move: <Term>Luma Bridge</Term> and the Adobe Commerce Optimizer Connector. Luma is Adobe's classic server-side Commerce storefront theme. If you run a <Term>Luma</Term> storefront today and are moving to Edge Delivery Services, Luma Bridge is a PHP module on Commerce that enables Luma pages and EDS drop-ins to share the same cart and sign-in session during the transition. For the full stack (drop-ins, boilerplate, and how services connect), see [Commerce services and backends](/get-started/architecture/commerce-services-and-backends/).
```

Install and configure the following on Commerce PaaS before you create your storefront. Adobe Commerce as a Cloud Service and Adobe Commerce Optimizer manage these automatically. Skip this section if you are not on Commerce PaaS.

### Minimum checklist before you create a storefront

Before <Term>Site Creator</Term>, manual configuration, or local preview can rely on Commerce APIs, confirm these items on Commerce PaaS:

1. You are on Adobe Commerce v2.4.7 or later (Magento Open Source is not supported for this storefront path).

> 1. You installed the <Term>Storefront Compatibility package</Term> ([Install guide](/setup/configuration/storefront-compatibility/install/)) so cart, checkout, account, and order GraphQL match what drop-ins expect.

Reviewer: It's not just checkout, account, and order. Maybe you should say the SCP fills the gaps in the core GraphQL schema that the drop-ins require.

1. You connected the storefront services your project needs (at minimum, plan for Services Connector and Catalog Service, then add Live Search, Product Recommendations, and Data Connection per your rollout). Use the [service documentation table](#storefront-services) below for links.

### Storefront Compatibility package

> Extends GraphQL for cart, checkout, account, and order flows that drop-ins expect. Overview: [Commerce services](/get-started/architecture/commerce-services-and-backends/#commerce-services). Install: [Install the Storefront Compatibility package](/setup/configuration/storefront-compatibility/install/) (v2.4.7 or v2.4.8).

Reviewer: This is repeating what you said a few lines ago,
